src/ai/backend/manager/models/session.py

```diff
@@ -1172,7 +1176,8 @@ class Meta:
     status_changed = GQLDateTime()
     status_info = graphene.String()
     status_data = graphene.JSONString()
-    status_history = graphene.JSONString()
+    status_history = graphene.JSONString()  # legacy
+    status_history_log = graphene.JSONString(description="Added in 24.09.0")
```

Comment:
Referring [our internal document](https://lablup.sharepoint.com/:fl:/g/contentstorage/CSP_93a3ec6b-3a47-4174-aba8-e022f64d2e6a/EUnXwjFPLWxJhso05OMe8jUB404OyCc_Vpm8hOremhVvKA?e=9C3ACs&nav=cz0lMkZjb250ZW50c3RvcmFnZSUyRkNTUF85M2EzZWM2Yi0zYTQ3LTQxNzQtYWJhOC1lMDIyZjY0ZDJlNmEmZD1iJTIxYS15amswYzZkRUdycU9BaTlrMHVhcnVadTBHN1ZEdEZwem9aRi1qaV91OFdsV0h0XzlhY1RyVEJBWHg0aDFZZiZmPTAxSzRWMkRLU0oyN0JEQ1RaTk5SRVlOU1JVNFRSUjU0UlYmYz0lMkYmYT1Mb29wQXBwJng9JTdCJTIydyUyMiUzQSUyMlQwUlRVSHhzWVdKc2RYQXVjMmhoY21Wd2IybHVkQzVqYjIxOFlpRmhMWGxxYXpCak5tUkZSM0p4VDBGcE9Xc3dkV0Z5ZFZwMU1FYzNWa1IwUm5CNmIxcEdMV3BwWDNVNFYyeFhTSFJmT1dGalZISlVRa0ZZZURSb01WbG1mREF4U3pSV01rUkxWVkpLVFVSTlUwSkxSbGRXUWpOSVZrZEpUVFZMVmtsU1RrVSUzRCUyMiUyQyUyMmklMjIlM0ElMjI4MTJhZTI0OS1lOTA0LTQ4YjEtOGQ1ZC04OWIxZjc0ZjAyMTQlMjIlN0Q%3D), add the `deprecation_reason` argument to the `status_history` field.